iChef-WebUi/src/app/components/header/header.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+import { HEADER_LINKS } from 'src/app/constants/header';
+import { BUTTON, LANDING_PAGE } from 'src/app/constants/texts';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+
+export class HeaderComponent {
+
+    readonly homeLink = HEADER_LINKS.HOME;
```

Comment:
See comments in landing page component

Reply:
Is it ok if we remove 'homeLink' and only use 'HEADER_LINKS.HOME' instead? (in the places where we needed 'homeLink'). Same applies for 'recipesLink'. Or we can remove 'HEADER_LINKS.HOME' and only use string constants, but yet again, it seemed more organized by using 'HEADER_LINKS'.

Reply:
The thing that bothers me is that we create a lot of readonly variables that just hold constants declared elsewhere. In my opinion we can use HEADER_LINKS but declaring a readonly variable in each component this constant is a [Code Smell](https://en.wikipedia.org/wiki/Code_smell#:~:text=In%20computer%20programming%2C%20a%20code,%2C%20developer%2C%20and%20development%20methodology.&text=It%20is%20also%20a%20term%20used%20by%20agile%20programmers.).